json/concatenated_json_parse_stream_test.ts

```diff
@@ -4,7 +4,7 @@ import { ConcatenatedJsonParseStream } from "./concatenated_json_parse_stream.ts
 import { assertInvalidParse, assertValidParse } from "./_test_common.ts";
 
 Deno.test({
-  name: "[json] ConcatenatedJsonParseStream",
+  name: "ConcatenatedJsonParseStream()",
```

Comment:
Can you please omit the round brackets on class names?

Reply:
done